atom/browser/api/atom_api_web_contents.cc

```diff
@@ -526,18 +527,12 @@ void WebContents::FindReply(content::WebContents* web_contents,
   v8::HandleScope handle_scope(isolate());
 
   mate::Dictionary result = mate::Dictionary::CreateEmpty(isolate());
-  if (number_of_matches == -1) {
-    result.Set("requestId", request_id);
-    result.Set("selectionArea", selection_rect);
-    result.Set("finalUpdate", final_update);
-    result.Set("activeMatchOrdinal", active_match_ordinal);
-    Emit("found-in-page", result);
-  } else if (final_update) {
-    result.Set("requestId", request_id);
-    result.Set("matches", number_of_matches);
-    result.Set("finalUpdate", final_update);
-    Emit("found-in-page", result);
-  }
+  result.Set("requestId", request_id);
```

Comment:
I was trying to match the old behavior since the behavior of `FindReply` has changed, I don't know why this event was emitting when `finalUpdate` is false before.